server/selinux/server/pulp-celery.te

```diff
@@ -40,8 +47,17 @@ allow celery_t self:unix_dgram_socket { create connect };
 #
 
 allow celery_t self:process { setsched };
-allow celery_t tmp_t:dir { write remove_name create add_name rmdir setattr };
-allow celery_t tmp_t:file { rename create unlink setattr write read getattr open lock };
+
+allow celery_t celery_tmp_t:dir { write remove_name create add_name rmdir setattr };
```

Comment:
I think this statement should be:

`allow celery_t celery_tmp_t:dir manage_dir_perms;`

I looked up what that macro expands into in `/usr/share/selinux/devel/include/support/obj_perm_sets.spt` and it says you'll get:

`{ create open getattr setattr read write link unlink rename search add_name remove_name reparent rmdir lock ioctl }`

You'll get the obj_perm_sets.spt file if you install `selinux-policy-devel`

This expands the permissions by adding `{ open getattr read link unlink rename search reparent lock ioctl }`, but for the tmp dirs context (`celery_tmp_t`) I think this is the right thing to do. This needs to be checked that it compiles on FC20, EL6, and EL7 to esnure this macro is defined in those versions of `selinux-policy-devel`.